src/gromo/growing_module.py

```diff
@@ -1183,32 +1183,58 @@ def reset_computation(self) -> None:
     def delete_update(
         self,
         include_previous: bool = True,
+        include_output: bool = False,
     ) -> None:
         """
         Delete the update of the optimal added parameters.
```

Comment:
```suggestion
        Delete the optimal parameters added for layer growth.
```

Reply:
I don't like the fact that we are talking about an update of optimal added parameters. The _update_ is the added parameters, because it is a growth update.